build.gradle

```diff
@@ -31,11 +34,21 @@ repositories {
 }
 
 dependencies {
-    testCompile 'org.junit:junit5-api:5.0.0-ALPHA'
+    testCompile "org.junit.jupiter:junit-jupiter-api:${junitJupiterVersion}"
+    testRuntime "org.junit.jupiter:junit-jupiter-engine:${junitJupiterVersion}"
+    testRuntime "org.apache.logging.log4j:log4j-core:${log4JVersion}"
+    testRuntime "org.apache.logging.log4j:log4j-jul:${log4JVersion}"
 }
 
-junit5 {
-    version '5.0.0-ALPHA'
+junitPlatform {
+    logManager 'org.apache.logging.log4j.jul.LogManager'
```

Comment:
Don't we need `platformVersion` here?  it is described in [junit5 user guide](http://junit.org/junit5/docs/current/user-guide/#running-tests-build) .